packages/repository/src/relations/has-one/has-one.repository.ts

```diff
@@ -87,4 +96,12 @@ export class DefaultHasOneRepository<
     }
     return found[0];
   }
+
+  async delete(where?: Where<TargetEntity>, options?: Options): Promise<Count> {
```

Comment:
Please add tests to cover this new repository method, see [`packages/repository/src/__tests__/acceptance/has-one.relation.acceptance.ts`](https://github.com/strongloop/loopback-next/blob/master/packages/repository/src/__tests__/acceptance/has-one.relation.acceptance.ts)

Reply:
ok